### ISSUE TITLE
security icon should display in title mode

### ISSUE THREAD

**Issue by bridiver** [brave/browser-laptop#3651]
**Did you search for similar issues before submitting this one?**
Yes

**Describe the issue you encountered:**
Title mode does not show the lock icon so it's hard to know if the site is secure or not without hovering over the urlbar

**Expected behavior:**
Lock icon should always display the current status of the page
- Platform (Win7, 8, 10? macOS? Linux distro?):
  all
- Brave Version:
  0.11.6
- Steps to reproduce:
  1. Go to any http site
  2. Lock icon only appears when hovering over the urlbar
- Screenshot if needed:
- Any related issues:
  #2178

**Comment by bradleyrichter** [comment]
do you mean colored?

![image](https://cloud.githubusercontent.com/assets/13509546/18187816/ae02f77a-7061-11e6-9602-f54a7e83f170.png)

**Comment by bridiver** [comment]
no, I was thinking about http sites. In normal mode it shows for both, but only shows the lock for https in title mode

**Comment by bradleyrichter** [comment]
@bridiver In my experience, it is more important to know if the site is secure. Right now with the grey icon color in title mode, showing both may be problematic for differentiation. 

I'll work an improvement into the new toolbar design changes though....

**Comment by bridiver** [comment]
sure, but I think it's good to draw attention to the lack of security as well

**Comment by bbondy** [comment]
I agree with that last point.
/cc @diracdeltas

**Comment by bradleyrichter** [comment]
We will have new icons for these, and they will all work better in both modes when we switch over to the left-align URL bar layout.

**Comment by diracdeltas** [comment]
we tried showing it in both locked and unlocked states at the beginning, but decided it was too visually noisy for most people (i think?)

**Comment by bridiver** [comment]
I saw something recently about either FF or Chrome doing something to display insecure sites. With httpse built-in it seems like showing insecure is more important than showing secure

**Comment by diracdeltas** [comment]
@bridiver yeah that was Chrome, we are planning on doing the same (showing a negative signal for insecure sites and not showing anything for secure sites)

**Comment by bsclifton** [comment]
Potential icons available (should be merged in)- see notes in https://github.com/brave/browser-laptop/issues/2178

**Comment by Liunkae** [comment]
+1 from https://twitter.com/ericlaw/status/798929401505140736 (from duplicate #5686)
